R/system.R

```diff
@@ -1347,15 +1356,19 @@ queryPostgres <- function(host, port, databaseName, username, password, numOfRow
 }
 
 #' @export
-queryAmazonAthena <- function(driver = "", region = "", authenticationType = "IAM Credentials", s3OutputLocation = "", user = "", password = "", additionalParams = "", query = "", numOfRows = 0, stringsAsFactors = FALSE, as.is = TRUE, ...){
-  if(!requireNamespace("RODBC")){stop("package RODBC must be installed.")}
+queryAmazonAthena <- function(driver = "", region = "", authenticationType = "IAM Credentials", s3OutputLocation = "", user = "", password = "", additionalParams = "", query = "", numOfRows = -1, stringsAsFactors = FALSE, as.is = TRUE, ...){
+  if(!requireNamespace("odbc")){stop("package RODBC must be installed.")}
   conn <- getAmazonAthenaConnection(driver = driver, region = region, authenticationType = authenticationType, s3OutputLocation = s3OutputLocation, user = user, password = password, additionalParams = additionalParams)
   tryCatch({
+    # For backwawrd compatibility, if 0 is passed as numOfRows, change it to -1.
```

Comment:
Add some more explanation about what backword compatibility this was about for record.

Reply:
Added more comment.